plugins/javadoc/src/main/resources/views/class.korte

```diff
@@ -38,6 +38,24 @@
 {% if signature.supertypes != null %}{{signature.supertypes|raw}}{% endif %}
                     </pre>
                     <div class="block">{{ classlikeDocumentation|raw }}</div>
+
+                    <dl>
+
+                    {% if sinceTagContent != "" %}
```

Comment:
Looks like this needs to be a size check, otherwise there are empty `Since` blocks:

<details>
<summary>screenshot</summary>

![2023-05-30_20-48-44](https://github.com/Kotlin/dokka/assets/22685910/7dac3982-3852-4133-9b67-06b418e2f049)

</details>


```suggestion
                    {% if sinceTagContent.size() != 0 %}
```